content/tutorials/get-ether-and-ocean-tokens.md

```diff
@@ -1,32 +1,40 @@
 ---
 title: Get ETH and OCEAN Tokens
-description: How to get tokens for each network deployment
+description: How to get tokens for each network
 ---
 
-### Overview
+## Overview
 
-In each network, you'll need ETH to pay for gas. Certain Ocean actions will need OCEAN tokens.
+In each network, you'll need ETH and OCEAN: ETH to pay for gas, and OCEAN for certain Ocean actions.
 
-The ETH and OCEAN in Rinkeby and local networks are not "real" ETH or OCEAN tokens of any value. They're simply stand-ins to be able to test. 
+Ethereum mainnet is a network for production settings. Accordingly, ETH and OCEAN tokens have "real" value.
```

Comment:
Setting `real` in quotes sounds like they are not actually real. Without going philosophical, to highlight some word a better pattern is to make it italic:

```suggestion
Ethereum mainnet is a network for production settings. Accordingly, ETH and OCEAN tokens have _real_ value.
```